frontend/src/components/Analysis/BarChart/TableViewModal.tsx

```diff
@@ -0,0 +1,99 @@
+import { Modal, Table } from "antd";
+import React from "react";
+
+const decimalPlaces = 3;
+
+interface Props {
+  title: string;
+  visible: boolean;
+  onClose: () => void;
+  systemNames: string[];
+  xValues: string[];
+  yValues: number[][];
+  xLabel?: string;
+  yLabel?: string;
+  yAxisMax?: number;
+  confidenceScoresList: number[][][];
+  numbersOfSamplesList: number[][];
+}
+
+export function TableViewModal({
+  title,
+  visible,
+  onClose,
+  systemNames,
+  xLabel,
+  yLabel,
+  yAxisMax,
+  xValues,
+  yValues,
+  confidenceScoresList,
+  numbersOfSamplesList,
+}: Props) {
+  yAxisMax = yAxisMax === undefined ? 1 : yAxisMax;
+  const formattedxAxisData = xValues.map((x) => x.replace("\n|\n", "-"));
```

Comment:
What do you mean by "everywhere"? In the whole website? I feel like the current "vertical" version is still useful in our barcharts (shown below). If it becomes "horizontal", it would become squishy. 
<img width="408" alt="截圖 2022-12-01 下午7 29 18" src="https://user-images.githubusercontent.com/36850051/205187518-3d7cc68f-e1c4-4f11-9a6e-d60fa32bc1d9.png">

But if the "vertical" version is only used when displaying barchart, maybe this can become the special case, whereas the "horizontal" version can become the norm. This could be a potential future PR?

Reply:
Sounds good! Let's keep it as it is for this PR then.